tests/StackExchange.Redis.Tests/SentinelFailover.cs

```diff
@@ -62,6 +66,15 @@ public async Task ManagedMasterConnectionEndToEndWithFailoverTest()
             var sw = Stopwatch.StartNew();
             SentinelServerA.SentinelFailover(ServiceName);
 
+            // There's no point in doing much for 10 seconds - this is a built-in delay of how Sentinel works.
+            // The actual completion invoking the replication of the former master is handled via
+            // https://github.com/redis/redis/blob/f233c4c59d24828c77eb1118f837eaee14695f7f/src/sentinel.c#L4799-L4808
+            // ...which is invoked by INFO polls every 10 seconds (https://github.com/redis/redis/blob/f233c4c59d24828c77eb1118f837eaee14695f7f/src/sentinel.c#L81)
+            // ...which is calling https://github.com/redis/redis/blob/f233c4c59d24828c77eb1118f837eaee14695f7f/src/sentinel.c#L2666
+            // However, the quicker iteration on INFO during an o_down does not apply here: https://github.com/redis/redis/blob/f233c4c59d24828c77eb1118f837eaee14695f7f/src/sentinel.c#L3089-L3104
+            // So...we're waiting 10 seconds, no matter what. Might as well just idle to be more stable.
+            await Task.Delay(TimeSpan.FromSeconds(10));
```

Comment:
if the server polls every 10s, there's a 50% chance it will be done after 5s (since we don't know when the interval starts); is there any merit in trying in a loop at a smaller fraction? or ... just wait 10s?

Reply:
meh, it is a test; ignore me; KISS